src/Fantomas.Core/CodePrinter.fs

```diff
@@ -2127,6 +2128,39 @@ let genFunctionNameWithMultilineLids (trailing: Context -> Context) (longIdent:
     | _ -> sepNone
     |> genNode parentNode
 
+let (|EndsWithDualListApp|_|) (config: FormatConfig) (appNode: ExprAppNode) =
+    if not config.ExperimentalStroustrupStyle then
+        None
+    else
+        let mutable otherArgs = ListCollector<Expr>()
```

Comment:
Does this need to be marked as mutable? Doesn't look like it's being reassigned anywhere

Reply:
Yes, it needs to be mutable for some reason.
It doesn't work without it and is listed as such in the [RFC](https://github.com/fsharp/fslang-design/blob/main/FSharp-6.0/FS-1099-list-collector.md).

Reply:
Huh, interesting, never seen that type before